hazelcast/src/main/java/com/hazelcast/config/XmlConfigBuilder.java

```diff
@@ -1726,6 +1726,21 @@ private RingbufferStoreConfig createRingbufferStoreConfig(Node node) {
         return config;
     }
 
+    private MergePolicyConfig createMergePolicyConfig(Node node) {
```

Comment:
This will be used by different data structures?

Reply:
Yes, I pulled out all commonly used methods into this PR. The usage of this snippet will be like:
```java
            } else if ("merge-policy".equals(nodeName)) {
                MergePolicyConfig mergePolicyConfig = createMergePolicyConfig(n);
                config.setMergePolicyConfig(mergePolicyConfig);
````